node-commons/src/main/java/org/apache/iotdb/commons/utils/ThriftCommonsSerDeUtils.java

```diff
@@ -81,6 +82,18 @@ public static TEndPoint deserializeTEndPoint(ByteBuffer buffer) {
     return endPoint;
   }
 
+  public static void serializeInt(int nodeId, DataOutputStream stream) {
+    try {
+      stream.writeInt(nodeId);
+    } catch (IOException e) {
+      throw new ThriftSerDeException("Write Int failed: ", e);
+    }
+  }
+
+  public static int deserializeInt(ByteBuffer buffer) {
+    return buffer.getInt();
+  }
+
```

Comment:
These lines of code can be removed since using `BasicStructureSerDeUtil` is enough~